styleHome.css

```diff
@@ -0,0 +1,440 @@
+a{
+    float: left;
+    text-decoration: none;
+    padding: 30px;
+	color: #347AAD;
+}
+body{
+    padding:0px;
+    margin:0px;
+	font-family:sans-serif;
+}
+*{
+    box-sizing: border-box;
+}
+.orxe-header{
+    height: 80px;
+	border-bottom:1px solid #000000;
+}
+.hovering{
+	display:flex;
+}
+.hovering ul{
+	display:flex;
+    list-style-type:none;
+	margin:0px;
+    padding:0px;
+}
+.orxe-logo{
+    float: left;  
+}
+.itinerary{
+    float: right;
+    width: 175px;
+    height: 80px;
+    background-color: #347AAD;
+	padding-top:8px;
+}
+.itinerary a{
+    text-align: center;
+    color: white;
+    padding-left: 35px;
+    font-size: 20px;
+    padding-top: 5px;
+}
+.arrow{
+	font-size:40px;
+	padding-bottom:0px;
+}	
+.orxe-right{
+    float: right;
+    display: inline-block;
+}
+img{	
+    height:60px;
+    width:210px;
+    position:relative;
+    right:33px;
+    bottom:5px;   
+}
+.current-user-details{
+    padding-top: 0px;
+    padding-left: 240px;
+}
+.user-details{
+    float: right;
+    padding-top: 12px;
+}
+.user-details a{
+    padding-left: 10px;
+    border-right: 1px solid #000000;
+    border-spacing: 2px;
+}
+.current-user p{
+    padding-top: 10px;
+    margin-top: 0px;
+    margin-right: 10px;
+}
+a{
+    padding: 0px;
+    margin: 0px;
+}
+p{
+    padding: 0px;
+    margin: 0px;
+}
+.active{
+    background-color: #347AAD;
+    color: white;
+	padding:20px;
+}
+.ruler{
+    padding: 0px;
+    margin: 0px;
+	width: 100%;
+	height:2px;
+}
+.hovering a:hover{
+    background-color: #347AAD;
+    color: white;
+	padding:20px;
+}
+.hovering a{
+    padding: 20px;
+}
+.no-border:last-child{
+    padding-right: 30px;
+    border-right: none;
+    outline: none;
+}
+.flex{
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+	padding-left:40px;
+	margin:0 auto;
+	width:74%;
+}
+.image{
+    width: 430px;
+	height:220px;
+    margin: 25px;
+    text-align: center;
+}
+.image:last-child{
+	margin-right:0px;
+}
+.container{
+	width:74%;
+	margin:0 auto;
+}
+.deal{
+	display: flex;
+    flex-direction: row;
+    justify-content: left;
+    align-items: center;
+    border: 1px solid #000000;
+    background-color: #5E5E5E;
+	width:74%;
+	padding:4px;
+	color:#FFFFFF;
+	margin:0 auto;
+	margin-top:5px;
+}
+.submit{
+	background:#347AAD;
+	color:#FFFFFF;
+	padding-left:15px;
+	padding-right:15px;
+	padding-bottom:8px;
+	padding-top:8px;
+	border-top:#FFFFFF;
+	border-left:#FFFFFF;
+	box-shadow:0 8px 16px 0 rgba(0,0,0,0.2), 0 6px 10px 0 rgba(0,0,0,0.19);
```

Comment:
Why this I told you in last assignment also

Reply:
i asked you can we use rgba for opacity and you said yes , there is no other option

Reply:
not about rgba. It is about the syntax you repeat it twice which is not correct